### ISSUE TITLE
NTR: Ultrasound: thyroid cysts (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8982]
New term request Ultrasound: thyroid cysts (elthat_140106045634): Cystic  UMLS: 0162299

**Comment by pnrobinson** [comment]
@MickeySegal
is this distinct from  [Thyroglossal cyst HP:0010518](https://hpo.jax.org/app/browse/term/HP:0010518)? What is the context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1522/ "In a French series, thyroid nodules and/or cysts were detected by ultrasonography in 13/20 individuals (65%) with BHDS; no medullary carcinoma or other thyroid carcinomas were detected."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000853
- label: Thyroid cyst
- parents: [Abnormal thyroid morphology (HP:0011772)](https://hpo.jax.org/app/browse/term/HP:0011772)
- pmid: 
- def: A fluid-filled sac or cavity that develops due to fluid accumulation within the thyroid tissue.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8982
- synonyms:
